libraries/bsg_manycore_eva.cpp

```diff
@@ -1001,8 +1001,8 @@ int default_eva_size(
 
 }
 
-const hb_mc_coordinate_t default_origin = {.x = HB_MC_CONFIG_VCORE_BASE_X,
-                                           .y = HB_MC_CONFIG_VCORE_BASE_Y};
+const hb_mc_coordinate_t default_origin = {.x = 0,
```

Comment:
Not sure what the right default is here (what is default_map used for)? Seems dangerous to have hardcoded origins as we see :)

Reply:
Oof. I would say -1 and -1 but coordinate is an unsigned int. Certainly not 0, 0, because that could lead to a silent failure.

Maybe INT_MAX

Reply:
The default map is the original EVA map for the manycore. 

This struct may not matter anymore. It may be unrealized legacy code...